### DISCUSSION TITLE
make a manu screen

### DISCUSSION THREAD

**Discussion by Quantaviousdingleberry420** [aeolus-1/triangleGame.github.io#61] [Ideas]
make a menu screen that has buttons like play and options where you can ajust music sounds and change the colours and enable/disable cheats.

**Comment by nekoify** [comment]
manu 💀💀💀💀💀

**Reply by serifonium** [reply]
hey guys quandale dingle here

**Reply by Quantaviousdingleberry420** [reply]
L + ratio + maidenless + touch grace

**Reply by nekoify** [reply]
touch grace 💀💀💀💀💀💀💀💀💀💀💀💀

**Comment by Quantaviousdingleberry420** [comment]
bruh i mispelt menu

**Comment by aeolus-1** [comment]
no
